### PR TITLE
platformio 5.1.1

### PR DESCRIPTION
**Pull request by ivankravets**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

* Fixed a "The command line is too long" issue with a linking process on Windows ([issue #3827](https://github.com/platformio/platformio-core/issues/3827))
* Fixed an issue with [device monitor](https://docs.platformio.org/page/core/userguide/device/cmd_monitor.html) when the "send_on_enter" filter didn't send EOL chars ([issue #3787](https://github.com/platformio/platformio-core/issues/3787))
* Fixed an issue with silent mode when unwanted data is printed to stdout ([issue #3837](https://github.com/platformio/platformio-core/issues/3837))
* Fixed an issue when code inspection fails with "Bad JSON" ([issue #3790](https://github.com/platformio/platformio-core/issues/3790))
* Fixed an issue with overriding user-specified debugging configuration information in VSCode ([issue #3824](https://github.com/platformio/platformio-core/issues/3824))

### CODE REVIEW

File: Formula/platformio.rb [hunk 1]
```diff
@@ -15,14 +14,18 @@ class Platformio < Formula
     sha256 cellar: :any_skip_relocation, mojave:        "c2d69c80339fae7316704012f520bfe140c3141d940b87eb1524dd92e742fc63"
   end
 
-  depends_on "python-tabulate"
```
Comment (carlocab) [inline Formula/platformio.rb:18]: Please don't remove this; this is equivalent to the `tabulate` resource.

File: Formula/platformio.rb [hunk 2]
```diff
@@ -98,6 +96,11 @@ class Platformio < Formula
     sha256 "7d49f4a27f8742262ef1470608c59ddbc66baf37c148e938c7038e6bc7a998aa"
   end
 
+  resource "tabulate" do
+    url "https://files.pythonhosted.org/packages/ae/3d/9d7576d94007eaf3bb685acbaaec66ff4cdeb0b18f1bf1f17edbeebffb0a/tabulate-0.8.9.tar.gz"
+    sha256 "eb1d13f25760052e8931f2ef80aaf6045a6cceb47514db8beab24cded16f13a7"
+  end
+
```
Comment (carlocab) [inline Formula/platformio.rb:103]:
```suggestion
```

**Review by carlocab — COMMENTED** [review]
Thanks for your PR, @ivankravets. One small comment.

**Comment by ivankravets** [comment]
Thanks! I'll open new PR/

**Pull request closed without merging**